modules/platforms/dotnet/Directory.Build.props

```diff
@@ -22,15 +22,31 @@
         <LangVersion>9</LangVersion>
         <Nullable>enable</Nullable>
         <Version>3.0.0-alpha3</Version>
+
+        <Company>Apache Software Foundation</Company>
+        <Authors>Apache Ignite</Authors>
+        <Copyright>2015 - 2021 The Apache Software Foundation</Copyright>
```

Comment:
Would we need to update the date manually every year? Is year necessary?

Reply:
Good point. I've looked around and it seems acceptable to omit the year in this section. Fixed.